consensus/expected.go

```diff
@@ -297,3 +306,40 @@ func (c *Expected) createPowerTableView(st state.Tree) PowerTableView {
 func (c *Expected) loadStateTree(ctx context.Context, id cid.Cid) (state.Tree, error) {
 	return state.LoadStateTree(ctx, c.cstore, id)
 }
+
+// verifyBLSMessageAggregate errors if the bls signature is not a valid aggregate of message signatures
+func verifyBLSMessageAggregate(sig types.Signature, msgs []*types.MeteredMessage) error {
```

Comment:
Does this work when there are no bls messages?  Does `VerifyBLSAggregate(nil, nil, nil)` evaluate to true?

Reply:
An empty signature does not validate against an empty slice of messages. Blocks will need to contain a valid `BLSAggregateSig` of a an empty list of bls messages (i.e. `bls.Aggregate([]bls.Signature{})`).